packer/install-dependencies.sh

```diff
@@ -313,9 +313,9 @@ for AZURE_CNI_NETWORKMONITOR_VERSION in ${AZURE_CNI_NETWORKMONITOR_VERSIONS}; do
 done
 
 AZURE_NPM_VERSIONS="
+1.0.31
```

Comment:
this is only adding the new version to the VHD build, not using it in code... I assume you also intended to use this new npm version by default?

Reply:
yup I would like to use it by default

Reply:
you also need to change https://github.com/Azure/aks-engine/blob/master/pkg/api/k8s_versions.go#L46 then

Reply:
let me check the previous pr again